### DISCUSSION TITLE
Query parameter surrounded by quotes cause InvalidCharacterError

### DISCUSSION THREAD

**Discussion by leovalen** [livewire/livewire#5260] [7. Bugs (archived)]
I'm building a search UI and encountered an issue keeping the query string in sync.

I've set the $queryString property to ['query'] to have Livewire update the query string whenever a new query is entered.

When the query property is surrounded by quotes (""), however, the quotes are not correctly escaped/encoded and the query string is not updated. Then, any subsequent update of the query property will crash the application with an "Unhandled Promise Rejection" error:

```Unhandled Promise Rejection: InvalidCharacterError: The string contains invalid characters.```

A query with quotes inside rather than surrounding the string works fine and is properly encoded.

Livewire 2.10.7 and Laravel 9.28.0.

**Comment by aisuvro** [comment]
save here
